src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                + "age="
```

Comment:
I would recommend to place each variable name on the same line with it's value for better readability.